python/ray/data/_internal/logical/operators/map_operator.py

```diff
@@ -62,6 +45,24 @@ def __init__(
         fn_constructor_kwargs: Optional[Dict[str, Any]] = None,
         ray_remote_args: Optional[Dict[str, Any]] = None,
     ):
+        """
+        Args:
+            name: Name for this operator. This is the name that will appear when
+                inspecting the logical plan of a Dataset.
+            input_op: The operator preceding this operator in the plan DAG. The outputs
+                of `input_op` will be the inputs to this operator.
+            block_fn: The transform function to apply to each input block to produce
```

Comment:
Shall we name this as `transform_fn`? It's consistent (and IMO more clear) with the naming in new backend.

Reply:
yeah, actually after second thought, this argument should not be part of logical operator. Logical operator should only care about user-defined function. This is internal function we generate for execution. Plan to remove it and put into `planner` directory in a separate PR.